tests/backup/backup_test.go

```diff
@@ -7075,6 +7075,134 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
 	})
 })
 
+// This testcase verifies schedule backup creation with a single namespace.
+var _ = Describe("{ScheduleBackupCreationSingleNS}", func() {
+	var contexts []*scheduler.Context
+	var appContexts []*scheduler.Context
+	var backupLocationName string
+	var backupLocationUID string
+	var cloudCredUID string
+	var bkpNamespaces []string
+	var cloudAccountName string
+	var backupName string
```

Comment:
can you declare one var and name all the variables in it
for ex:

var (
contexts []*scheduler.Context
appContexts []*scheduler.Context
backupLocationName string
)

Reply:
Updated